portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteAppsUpgrade.testcase

```diff
@@ -0,0 +1,45 @@
+@component-name = "portal-upgrades"
+definition {
+
+	property database.types = "db2,mariadb,mysql,oracle,postgresql,sqlserver,sybase";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "Remote Apps";
+	property testray.main.component.name = "Upgrades Remote Apps";
+
+	setUp {
+		SignIn.signIn();
+
+		var portalURL = PropsUtil.get("portal.url");
+
+		AssertLocation.assertPartialLocation(value1 = "${portalURL}/web/guest");
+
+		SearchAdministration.executeReindex();
+	}
+
+	@description = "Upgrade to 7.3.10.1 and assert remote app fields were saved properly"
+	@priority = "5"
+	test CanViewRemoteAppFieldsAfterUpgrade73101 {
+		property portal.acceptance = "true";
+		property data.archive.type = "data-archive-remote-app";
+		property database.types = "db2,mariadb,mysql,oracle,postgresql,sybase";
```

Comment:
Remove line. Don't need to set `property database.types = "db2,mariadb,mysql,oracle,postgresql,sybase";` in both global and testcase level.

Reply:
oh i see thanks!! Happy (early) New Years John!! :tada: